platform/broadcom/sonic-platform-modules-dell/s6000/scripts/platform_reboot_override

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+
+import os
+import struct
+
+PORT_RES = '/dev/port'
+COLD_RESET = 0xE     # Cold Reset 
+WARM_RESET = 0x6     # Warm Reset 
+RESET_REG = 0xCF9
+
+def io_reg_write(resource, offset, val):
+    fd = os.open(resource, os.O_RDWR)
+    if(fd < 0):
```

Comment:
Parentheses unnecessary. Please remove.

Reply:
Thank You. I have used the existing platform_reboot script (which has been removed as part of this PR) and edit it accordingly. Will do the changes and update the PR.

Reply:
@rkdevi27 are you going to update this PR?

Reply:
> @rkdevi27 are you going to update this PR?

@sujinmkang  Updated the PR. Thanks.